content/non-web-ict.md

```diff
@@ -55,47 +57,39 @@ Quick link: [WCAG2ICT Note](https://www.w3.org/TR/wcag2ict/)
--   **Comments on the definitions** in the WCAG 2.0 Glossary.
+-   **Key Terms** related to applying WCAG 2 to non-web documents and software.
+-   **Comments on the definitions** in the WCAG 2 Glossary.
 -   **Comments on conformance**.
 -   **Background information** on some topics.
 
-WCAG2ICT includes material from the WCAG 2.0 standard to provide context, along with specific guidance related to non-web ICT, formatted as follows:
+WCAG2ICT includes material from the WCAG 2 standard to provide context, along with specific guidance related to non-web ICT, formatted as follows:
 
--   **WCAG 2.0 principles, guidelines, and success criteria** — the exact text from the WCAG 2.0 standard. *These are visually styled in pale yellow boxes and usually prefaced with "Principle...", "From Guideline...", or "From Success Criterion..".*
--   **Excerpted text from the "Intent" sections of Understanding WCAG 2.0**, an informative supporting document. *These are visually styled in pale yellow boxes and prefaced with "Intent from Understanding Success Criterion...".*
+-   **WCAG 2 principles, guidelines, and success criteria** — the exact text from the WCAG 2 standard. *These are visually styled in pale yellow boxes and usually prefaced with "Principle...", "From Guideline...", or "From Success Criterion..".*
+-   **Excerpted text from the "Intent" sections of Understanding WCAG 2**, an informative supporting document. *These are visually styled in pale yellow boxes and prefaced with "Intent from Understanding Success Criterion...".*
 -   **Guidance on applying each success criteria to non-web documents and software**. *These are visually styled in pale blue boxes with a heading on a dark blue background that starts with "Additional Guidance...".*
 
 ### Technical document format
 
-WCAG2ICT follows the [W3C](https://www.w3.org) format for technical
-reports, which has several sections at the beginning, including links to
-different versions, editors, abstract, and status.
+WCAG2ICT follows the [W3C](https://www.w3.org) format for technical reports, which has several sections at the beginning, including links to different versions, editors, abstract, and status.
 
 ## Who develops WCAG2ICT {#wg}
 
-WCAG2ICT was developed by the [WCAG2ICT Task Force](https://www.w3.org/WAI/GL/WCAG2ICT-TF/), provided for public review, and approved by the Web Content Accessibility Guidelines Working Group ([WCAG WG](https://www.w3.org/WAI/GL/)), which is part of the World Wide Web Consortium ([W3C](https://www.w3.org/)) Web Accessibility Initiative ([WAI](https://www.w3.org/WAI/)).
+WCAG2ICT is developed by the [WCAG2ICT Task Force](https://www.w3.org/WAI/GL/WCAG2ICT-TF/), provided for public review, and approved by the Web Content Accessibility Guidelines Working Group ([WCAG WG](https://www.w3.org/WAI/GL/)), which is part of the World Wide Web Consortium ([W3C](https://www.w3.org/)) Web Accessibility Initiative ([WAI](https://www.w3.org/WAI/)).
```

Comment:
The WCAG WG has changed to Accessibility Guidelines Working group (AG WG), so the info here should match, right?  Not sure how you typically handle these WG name changes.

Reply:
thanks for the catch.